adcm_aio_client/core/objects/cm.py

```diff
@@ -110,3 +120,78 @@ def get_own_path(self: Self) -> Endpoint:
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class Prototype[ParentObject: InteractiveObject](InteractiveChildObject[ParentObject]):
```

Comment:
There is no such entity in the requirements. We don't want the user to know about the existence of such an entity. Please delete it

Reply:
deleted